apps/apple/FluentUITester/macos/FluentUITester-macOS/AppDelegate.m

```diff
@@ -0,0 +1,32 @@
+#import "AppDelegate.h"
+
+#import <React/RCTBridge.h>
+#import <React/RCTBundleURLProvider.h>
+
+@interface AppDelegate () <RCTBridgeDelegate>
+
+@end
+
+@implementation AppDelegate
+
+- (void)awakeFromNib {

```

Comment:
No need for awakeFromNib - just do this in applicationDidFinishLaunching? Important to be precise with this as we'd expect many clients to copy it